api/config/config_test.go

```diff
@@ -478,6 +514,12 @@ func TestValidate(t *testing.T) {
 				UI: &config.UIConfig{
 					ProjectInfoUpdateEnabled: true,
 				},
+				UpdateProject: &config.UpdateProjectConfig{
+					Endpoint:         "http://example-update-project.dev",
+					PayloadTemplate:  "your-payload-template",
+					ResponseTemplate: "your-response-template",
+					LabelsBlacklist:  []string{},
+				},
```

Comment:
Can we add additional test cases that actually test the behaviour of `Endpoint         string `validate:"url"`` that you added to `config.go`?